aten/src/ATen/TensorIterator.cpp

```diff
@@ -300,7 +300,7 @@ void TensorIteratorBase::compute_types(const TensorIteratorConfig& config) {
 
   // Promotes common dtype to the default float scalar type, if needed
   if (config.promote_integer_inputs_to_float_ &&
-      c10::isIntegralType(common_dtype_, /*include_bool=*/true)) {
+      c10::isIntegralType(common_dtype_, /*includeBool=*/true)) {
```

Comment:
what is the significance of this change?

Reply:
It's sort of named-arg clang-tidy calling convention.
I.e. since isIntegralType is defined as  `:static inline bool isIntegralType(ScalarType t, bool includeBool) {`), `clang-tidy` raises an auto-fixeable warning, that comment format should match the declared variable name.